x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/RestSqlQueryAction.java

```diff
@@ -26,10 +28,13 @@
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 import static org.elasticsearch.rest.RestRequest.Method.POST;
+import static org.elasticsearch.xpack.ql.util.LoggingUtils.wrapWithFailureLogging;
 import static org.elasticsearch.xpack.sql.proto.CoreProtocol.URL_PARAM_DELIMITER;
 
 @ServerlessScope(Scope.PUBLIC)
 public class RestSqlQueryAction extends BaseRestHandler {
+    private static final Logger LOGGER = LogManager.getLogger(RestSqlQueryAction.class);
+    private static final String LOGGING_PREFIX = "SQL query";
```

Comment:
nit:
```suggestion
    private static final String LOGGING_PREFIX = "SQL request";
```
otherwise it might be confusing if a `_sql/translate` fails.